src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page" >
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page__body" >
+    <div class="card" data-qa="card">
+
+      <img class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac">
```

Comment:
```suggestion
      <img class="card__image"
        src="./images/imac.jpeg"
        alt="imac"
      >
```
A closing bracket should be on the same level as an opening bracket (if you have multiline syntax)

Reply:
Got you, thanks! Will change now ;)